Formula/erlang.rb

```diff
@@ -8,6 +8,7 @@ class Erlang < Formula
 
   bottle do
     cellar :any
+    rebuild 1
```

Comment:
Yeah, I'm just not sure why you're adding `rebuild` here manually. That'll get done by CI. There's no massive harm to adding it IIRC, but I was wondering if you were trying to bump the formula revision & misunderstood placement or such, was all 😃